mne/io/reference.py

```diff
@@ -193,11 +197,22 @@ def add_reference_channels(inst, ref_channels, copy=True):
                      'unit': FIFF.FIFF_UNIT_V,
                      'coord_frame': FIFF.FIFFV_COORD_HEAD,
                      'loc': np.zeros(12)}
+        # see if "zeroth" EEG electrode present in dig points
+        dig_loc = [dl for dl in inst.info['dig'] if (
+                   dl['kind'] == FIFF.FIFFV_POINT_EEG and
+                   dl['ident'] == 0)]
+        if len(dig_loc) > 1:  # is either 0 or 1, never >1 for Neuromag
+            warn('Multiple reference channel locations detected! Their '
+                 'location information will be ignored.')
+        else:
+            chan_info['loc'][:3] = dig_loc[0]['r'][:]
+            chan_info['loc'][3:6] = dig_loc[0]['r'][:]  # 3:6 holds ref loc
```

Comment:
It was an easy addition, so I implemented it. But the point of CAR is that the physical location of Ref is immaterial. I'll dig into this myself.